[{"type":"message","text":"\u003chttps://github.com/vim/vim/commit/cf4d454df0619ee41ef40e7e91fce3fb061d7d5b|patch 8.2.1920: listlbr test fails when run after another test\u003e\npatch 8.2.1920: listlbr test fails when run after another test\n\nProblem:    Listlbr test fails when run after another test.\nSolution:   Add test separately to list of test targets.","ts":"1603922280.038100","attachments":[{"fallback":"GitHub: patch 8.2.1920: listlbr test fails when run after another test · vim/vim@cf4d454","id":1,"title":"patch 8.2.1920: listlbr test fails when run after another test · vim/vim@cf4d454","title_link":"https://github.com/vim/vim/commit/cf4d454df0619ee41ef40e7e91fce3fb061d7d5b","text":"Problem: Listlbr test fails when run after another test. Solution: Add test separately to list of test targets.","thumb_url":"https://avatars0.githubusercontent.com/u/11618545?s=400\u0026v=4","blocks":null,"service_name":"GitHub","service_icon":"https://a.slack-edge.com/80588/img/unfurl_icons/github.png","thumb_width":48,"thumb_height":48,"from_url":"https://github.com/vim/vim/commit/cf4d454df0619ee41ef40e7e91fce3fb061d7d5b","original_url":"https://github.com/vim/vim/commit/cf4d454df0619ee41ef40e7e91fce3fb061d7d5b"}],"edited":{"user":"BPV6T0C1J","ts":"1603922281.000000"},"subtype":"bot_message","bot_id":"BPV6T0C1J","username":"Recent Commits to vim:master","icons":{"image_48":"https://a.slack-edge.com/80588/img/services/rss_48.png"},"replace_original":false,"delete_original":false,"blocks":null},{"type":"message","text":"\u003chttps://github.com/vim/vim/commit/28ee892ac4197421b3317f195512ca64cc56a5b4|patch 8.2.1919: assert_fails() setting emsg_silent changes normal exe…\u003e\npatch 8.2.1919: assert_fails() setting emsg_silent changes normal execution\n\nProblem:    Assert_fails() setting emsg_silent changes normal execution.\nSolution:   Use a separate flag in_assert_fails.","ts":"1603913886.037900","attachments":[{"fallback":"GitHub: patch 8.2.1919: assert_fails() setting emsg_silent changes normal exe… · vim/vim@28ee892","id":1,"title":"patch 8.2.1919: assert_fails() setting emsg_silent changes normal exe… · vim/vim@28ee892","title_link":"https://github.com/vim/vim/commit/28ee892ac4197421b3317f195512ca64cc56a5b4","text":"…cution Problem: Assert_fails() setting emsg_silent changes normal execution. Solution: Use a separate flag in_assert_fails.","thumb_url":"https://avatars0.githubusercontent.com/u/11618545?s=400\u0026v=4","blocks":null,"service_name":"GitHub","service_icon":"https://a.slack-edge.com/80588/img/unfurl_icons/github.png","thumb_width":48,"thumb_height":48,"from_url":"https://github.com/vim/vim/commit/28ee892ac4197421b3317f195512ca64cc56a5b4","original_url":"https://github.com/vim/vim/commit/28ee892ac4197421b3317f195512ca64cc56a5b4"}],"edited":{"user":"BPV6T0C1J","ts":"1603913888.000000"},"subtype":"bot_message","bot_id":"BPV6T0C1J","username":"Recent Commits to vim:master","icons":{"image_48":"https://a.slack-edge.com/80588/img/services/rss_48.png"},"replace_original":false,"delete_original":false,"blocks":null},{"type":"message","text":"\u003chttps://github.com/vim/vim/commit/3e2534ed1a6557445747e6c1439ad26ec8eabfc4|patch 8.2.1918: Vim9: E1100 mentions :let\u003e\npatch 8.2.1918: Vim9: E1100 mentions :let\n\nProblem:    Vim9: E1100 mentions :let.\nSolution:   Mention \"var\". (closes #7207)","ts":"1603905484.037700","attachments":[{"fallback":"GitHub: patch 8.2.1918: Vim9: E1100 mentions :let · vim/vim@3e2534e","id":1,"title":"patch 8.2.1918: Vim9: E1100 mentions :let · vim/vim@3e2534e","title_link":"https://github.com/vim/vim/commit/3e2534ed1a6557445747e6c1439ad26ec8eabfc4","text":"Problem: Vim9: E1100 mentions :let. Solution: Mention \"var\". (closes #7207)","thumb_url":"https://avatars0.githubusercontent.com/u/11618545?s=400\u0026v=4","blocks":null,"service_name":"GitHub","service_icon":"https://a.slack-edge.com/80588/img/unfurl_icons/github.png","thumb_width":48,"thumb_height":48,"from_url":"https://github.com/vim/vim/commit/3e2534ed1a6557445747e6c1439ad26ec8eabfc4","original_url":"https://github.com/vim/vim/commit/3e2534ed1a6557445747e6c1439ad26ec8eabfc4"}],"edited":{"user":"BPV6T0C1J","ts":"1603905484.000000"},"subtype":"bot_message","bot_id":"BPV6T0C1J","username":"Recent Commits to vim:master","icons":{"image_48":"https://a.slack-edge.com/80588/img/services/rss_48.png"},"replace_original":false,"delete_original":false,"blocks":null},{"type":"message","text":"\u003chttps://github.com/vim/vim/commit/159563b439d1b2a61d1003430b6c9f2a8509b5c5|patch 8.2.1917: no test for improved Man command\u003e\npatch 8.2.1917: no test for improved Man command\n\nProblem:    No test for improved Man command.\nSolution:   Test that shell arguments are properly escaped.","ts":"1603903080.037500","attachments":[{"fallback":"GitHub: patch 8.2.1917: no test for improved Man command · vim/vim@159563b","id":1,"title":"patch 8.2.1917: no test for improved Man command · vim/vim@159563b","title_link":"https://github.com/vim/vim/commit/159563b439d1b2a61d1003430b6c9f2a8509b5c5","text":"Problem: No test for improved Man command. Solution: Test that shell arguments are properly escaped.","thumb_url":"https://avatars0.githubusercontent.com/u/11618545?s=400\u0026v=4","blocks":null,"service_name":"GitHub","service_icon":"https://a.slack-edge.com/80588/img/unfurl_icons/github.png","thumb_width":48,"thumb_height":48,"from_url":"https://github.com/vim/vim/commit/159563b439d1b2a61d1003430b6c9f2a8509b5c5","original_url":"https://github.com/vim/vim/commit/159563b439d1b2a61d1003430b6c9f2a8509b5c5"}],"edited":{"user":"BPV6T0C1J","ts":"1603903081.000000"},"subtype":"bot_message","bot_id":"BPV6T0C1J","username":"Recent Commits to vim:master","icons":{"image_48":"https://a.slack-edge.com/80588/img/services/rss_48.png"},"replace_original":false,"delete_original":false,"blocks":null},{"type":"message","text":"\u003chttps://github.com/vim/vim/commit/171fb923b8f8da9fb0db1c8c86e35cf4e1339000|patch 8.2.1916: Vim9: function call is aborted even when \"silent!\" is…\u003e\npatch 8.2.1916: Vim9: function call is aborted even when \"silent!\" is used\n\nProblem:    Vim9: function call is aborted even when \"silent!\" is used.\nSolution:   Use did_emsg instead of called_emsg. (closes #7213)","ts":"1603901883.037300","attachments":[{"fallback":"GitHub: patch 8.2.1916: Vim9: function call is aborted even when \"silent!\" is… · vim/vim@171fb92","id":1,"title":"patch 8.2.1916: Vim9: function call is aborted even when \"silent!\" is… · vim/vim@171fb92","title_link":"https://github.com/vim/vim/commit/171fb923b8f8da9fb0db1c8c86e35cf4e1339000","text":"… used Problem: Vim9: function call is aborted even when \"silent!\" is used. Solution: Use did_emsg instead of called_emsg. (closes #7213)","thumb_url":"https://avatars0.githubusercontent.com/u/11618545?s=400\u0026v=4","blocks":null,"service_name":"GitHub","service_icon":"https://a.slack-edge.com/80588/img/unfurl_icons/github.png","thumb_width":48,"thumb_height":48,"from_url":"https://github.com/vim/vim/commit/171fb923b8f8da9fb0db1c8c86e35cf4e1339000","original_url":"https://github.com/vim/vim/commit/171fb923b8f8da9fb0db1c8c86e35cf4e1339000"}],"edited":{"user":"BPV6T0C1J","ts":"1603901884.000000"},"subtype":"bot_message","bot_id":"BPV6T0C1J","username":"Recent Commits to vim:master","icons":{"image_48":"https://a.slack-edge.com/80588/img/services/rss_48.png"},"replace_original":false,"delete_original":false,"blocks":null}]
